### DISCUSSION TITLE
Incorrect "gun runner" talent translation in Russian

### DISCUSSION THREAD

**Discussion by CAMOBAP18** [FakeFishGames/Barotrauma#14064] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

The "Gun Runner" talent should be translated into Russian as "Вы ходите на 20% быстрее при прицеливании. Вы переносите боеприпасы для орудий на 25% быстрее." or "Вы переносите боеприпасы для орудий на 25% быстрее. Вы ходите на 20% быстрее при прицеливании."

### Reproduction steps

Character tab "talent tree"

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! We've fixed the issue now, the fix should be included in the next update.
